Tests/StencilSwiftKitTests/XCTest+Assertions.swift

```diff
@@ -0,0 +1,12 @@
+//
+// StencilSwiftKit
+// Copyright (c) 2017 SwiftGen
+// MIT Licence
+//
+
+import XCTest
+
+public func XCTAssertThrows(_ expression: @autoclosure () throws -> Any?, _ message: @autoclosure () -> String = "",
```

Comment:
I remember discussing this, but for now we use `XCTAssertThrowsError` when we want / expect the expression to fail with an exception.